airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/__init__.py

```diff
@@ -139,13 +147,13 @@
         job=generate_oss_registry,
         resources_def=REGISTRY_ENTRY_RESOURCE_TREE,
         gcs_blobs_resource_key="latest_oss_registry_entries_file_blobs",
-        interval=30,
+        interval=60,
```

Comment:
Are we intentionally slowing things down here? What's the reasoning for doing so (just wondering)

Reply:
Snuck it in to deal with a senor resource issue.

https://airbytehq-team.slack.com/archives/C056HGD1QSW/p1690931225565009

Very minor change, figured I would Save an approve-and-merge.